components/pages/submission-system/donor/ClinicalTimeline/Timeline.tsx

```diff
@@ -109,7 +115,7 @@ const TimelineItem = ({ item, active, onClick, disabled }: TimeLineItemProps) =>
               justify-content: space-between;
             `}
           >
-            <Typography variant="caption" as="div">
+            <Typography variant="caption" as="div" bold={true}>
```

Comment:
```suggestion
            <Typography variant="caption" as="div" bold>
```

Removing `={true}` should work.